Add AGENTS.md and link OpenHands repo guidance (#340)
All-Hands-AI/OpenHands-CLI: AGENTS.md

- For UI/TUI changes, snapshot tests are the preferred evidence. If snapshots aren't available/appropriate, include screenshots (and note the terminal size used).
- Before opening a PR, run this verification flow (and include the exact commands run in the PR description):
  1. `make lint`
  2. `make test`
  3. If you touched ACP / end-to-end UI flow code (e.g., `e2e_tests/`, `openhands_cli/acp_impl/`, `openhands_cli/mcp/`, auth/connection flow): `uv run pytest e2e_tests`
  4. If you touched TUI code (e.g., `openhands_cli/tui/`, widgets, styles, layout): `uv run pytest tests/snapshots -v` (use `--snapshot-update` only for intentional UI changes)

#### PR submission checklist
- [ ] Scope is minimal and focused on one change
- [ ] Tests added/updated for behavior changes (or PR explains why not)
- [ ] `make lint`
- [ ] `make test`
- [ ] (If ACP/UI flow touched) `uv run pytest e2e_tests`
- [ ] (If TUI touched) snapshot tests run and snapshots updated/reviewed
- [ ] PR description includes: what changed, why, commands run, and UI evidence (snapshots/screenshots)

> 

Author: ```suggestion
```

# Repository Guidelines

## Repository Purpose
OpenHands CLI is a standalone terminal interface (Textual TUI) for interacting with the OpenHands agent.

This repo contains the current CLI UX, including the Textual TUI and a browser-served view via `openhands web`.

> The CLI originated as a port from the main OpenHands repository (CLI code under `openhands/cli/` in that repository) and was refactored to use the OpenHands agent-sdk (`openhands-sdk`, `openhands-tools`). Upstream OpenHands can still be a useful reference for behavior parity and shared concepts.

Author: ```suggestion
The CLI originated as a port from the main OpenHands repository and was rebuilt based on the OpenHands agent-sdk (`openhands-sdk`, `openhands-tools`). Upstream OpenHands can still be a useful reference for behavior parity and shared concepts.
```
